srcpkgs/linux-firmware/linux-firmware-amd.INSTALL

```diff
@@ -0,0 +1,7 @@
+# Regenerate initramfs.
+case ${ACTION} in
+post)
+	echo "Regenerating initramfs, please wait..."
+	dracut -f -q --regenerate-all
```

Comment:
Nope, this won't work. Dracut is not a mandatory part of void.

Reply:
What would you suggest instead?

Reply:
https://github.com/void-linux/void-packages/blob/master/Manual.md#kernel-hooks

Reply:
Hm no the kernel hooks only work for one version at a time.
I guess we would need a new xbps-triggers script, which supports all initram packages or a similiar directory like `/etc/initram.d` where the packages drop their regeneration hooks.